custom-tab/src/App.jsx

```diff
@@ -0,0 +1,57 @@
+import TabPage from "./pages/TabPage";
+
+function App() {
+  const data = [
```

Comment:
Ben data yi kendi file icine alirdim. data.jsx gibi. Component in icinde tutmazdim..

Reply:
Ayrica bu datanin usememo icine alinmasi lazim degil mi? Belki de degil, bir arastir bakalim